streamer/src/nonblocking/quic.rs

```diff
@@ -79,6 +82,9 @@ const CONNECTION_CLOSE_REASON_TOO_MANY: &[u8] = b"too_many";
 /// Limit to 250K PPS
 pub const DEFAULT_MAX_STREAMS_PER_MS: u64 = 250;
 
+pub const DEFAULT_MAX_CONNECTIONS_PER_IPADDR_PER_MINUTE: u32 = 8;
+const TOTAL_CONNECTIONS_PER_SECOND: u32 = 2500;
```

Comment:
is this 2500 the same as the sum of staked and unstaked connecions?

Reply:
yes. That was the idea. But it does not need to be exactly the same. We may evaluate and choose a different number in the future.